src/requirements/requirement-frontend-computation.ts

```diff
@@ -105,20 +105,14 @@ const getTotalCreditsFulfillmentStatistics = (
   }
 
   let minCountFulfilled = 0;
-  let minCountRequired = 120;
-  const courseCodeSet = new Set<string>();
+  const minCountRequired = 120;
   const eligibleCourses =
     college === 'AG'
       ? courses.filter(course => !course.code.startsWith('PE '))
       : courses.filter(courseIsAllEligible);
```

Comment:
I don't think we need to special case CALS here and we can just have `const eligibleCourses = courses.filter(courseIsAllEligible`  since it seems that courses that are 10** level or PE shouldn't count. 

Screenshot is of the [CALS graduation reqs](https://cals.cornell.edu/undergraduate-students/student-services/degree-requirements/graduation-requirements#:~:text=Minimum%20total%20credits%3A%20120%20academic,required%20for%20full%2Dtime%20status.)
![Screen Shot 2021-04-14 at 3 41 11 PM](https://user-images.githubusercontent.com/54298311/114769168-d78c7280-9d37-11eb-9c37-58949d2a9fd3.png)